frontend/src/features/crawl-workflows/workflow-editor.ts

```diff
@@ -1420,6 +1421,17 @@ https://archiveweb.page/images/${"logo.svg"}`}
         </btrix-language-select>
       `)}
       ${this.renderHelpTextCol(infoTextStrings["lang"])}
+      ${inputCol(html`
```

Comment:
We discussed this in Discord but to have it here: if a profile was created with a particular proxy, we likely want to either go ahead and set this automatically to the same proxy or at least display a recommendation to the user that they do the same. Users should have the ability to change or unselect a proxy if they want, but we should guide them toward the happy path.

Reply:
In the case where there's an org default proxy but a profile uses a different one, should probably set it to the latter?

Reply:
There's this `Check Profile` url under the profile dropdown, could fit a little yellow? warning text left to it as well, when profile proxy != workflow proxy and clicking it sets the workflow proxy to profile proxy.
![image](https://github.com/user-attachments/assets/0422c6c7-36ab-43ce-854b-87ca566d9fb9)

Reply:
We decided in conversations to not worry about this at this stage, and to make incremental improvements like this in time as we roll out proxies further.